File: 215-kth-largest-element-in-an-array_1.cpp
class Solution {
 public:
  int findKthLargest(vector<int>& nums, int k) {
    sort(nums.rbegin(), nums.rend());
    return nums[k - 1];
  }
};

/* Runtime: 12 ms, faster than 28.47% of C++ online submissions for Kth Largest Element in an Array.
 * Memory Usage: 10 MB, less than 83.39% of C++ online submissions for Kth Largest Element in an Array.
 */

File: 215-kth-largest-element-in-an-array_1.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        nums.sort(reverse=True)
        return nums[k - 1]

'''
Runtime: 82 ms, faster than 26.86% of Python3 online submissions for Kth Largest Element in an Array.
Memory Usage: 15.2 MB, less than 47.45% of Python3 online submissions for Kth Largest Element in an Array.
'''

